l2lidar_node:
  ros__parameters:

    # ---------------- Network ----------------
    l2_ip: "192.168.12.62"
    l2_port: 6101

    host_ip: "192.168.12.66"
    host_port: 6201

    # ---------------- Time Sync ----------------
    enable_l2_time_correction: true
    enable_l2_host_sync: true

    # sync rate host -> L2 (milliseconds)
    l2_sync_rate_ms: 50

    # ---------------- Latency ----------------
    enable_latency_measure: false

    # ---------------- Frames ----------------
    frame_id: "l2lidar_frame"
    imu_frame_id: "l2lidar_imu"

    # ---------------- Point Cloud ----------------
    frame3d: true
    imu_adjust: true

    # ---------------- ROS QoS ----------------
    pointcloud_queue_size: 10
    imu_queue_size: 10